lib/packExternalModules.js

```diff
@@ -9,7 +9,7 @@ const isBuiltinModule = require('is-builtin-module');
 const Packagers = require('./packagers');
 
 function rebaseFileReferences(pathToPackageRoot, moduleVersion) {
-  if (/^(?:file:[^/]{2}|\.\/|\.\.\/)/.test(moduleVersion)) {
+  if (/^(file:[^/]|\.\/|\.\.\/)/.test(moduleVersion)) {
```

Comment:
Should there be a test to make sure `file:./` works?

Reply:
Good idea!

Reply:
I tried to add some tests but there were all failed with the actual changes 🤔